src/compiler/bundle/bundle-output.ts

```diff
@@ -96,6 +96,9 @@ export const getRollupOptions = (
   const afterPlugins = config.rollupPlugins.after || [];
   const rollupOptions: RollupOptions = {
     input: bundleOpts.inputs,
+    output: {
+      inlineDynamicImports: bundleOpts.inlineDynamicImports,
```

Comment:
This code is also reached by the `worker-plugin` and `lazy-output` - how does that affect those two code paths?

Reply:
I might be misunderstanding what you're asking, but:
1. Rollup just moved the option to the `output` section, behavior doesn't change
2. Even though we previously didn't have one before, I added a `false` fallback value for the option if it's not defined on the passed `BundleOptions`. This will ensure there is always a value provided